src/components/FileMetadata/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+import { VersionFile } from '../../reducers/versions';
+import styles from './styles.module.scss';
```

Comment:
If this file does not contain anything, we can omit it I believe.

Reply:
How can we omit it and also use things like `className={styles.FileMetadata}`? I can hardcode the classNames instead, but you had requested I not do that in an earlier patch.

Reply:
heh, nevermind.